apps/web/src/components/Shared/Navbar/MobileDrawerMenu.tsx

```diff
@@ -0,0 +1,144 @@
+import { CheckCircleIcon, SwitchHorizontalIcon, XIcon } from '@heroicons/react/outline';
+import { Analytics } from '@lib/analytics';
+import formatHandle from '@lib/formatHandle';
+import getAvatar from '@lib/getAvatar';
+import isGardener from '@lib/isGardener';
+import isStaff from '@lib/isStaff';
+import { Trans } from '@lingui/macro';
+import type { Profile } from 'lens';
+import Link from 'next/link';
+import { useAppPersistStore, useAppStore } from 'src/store/app';
+import { PROFILE } from 'src/tracking';
+
+import Slug from '../Slug';
+import AppVersion from './NavItems/AppVersion';
+import Contact from './NavItems/Contact';
+import Logout from './NavItems/Logout';
+import Mod from './NavItems/Mod';
+import ReportBug from './NavItems/ReportBug';
+import Settings from './NavItems/Settings';
+import StaffMode from './NavItems/StaffMode';
+import Status from './NavItems/Status';
+import ThemeSwitch from './NavItems/ThemeSwitch';
+import YourProfile from './NavItems/YourProfile';
+
+const MobileDrawerMenu = () => {
+  const profiles = useAppStore((state) => state.profiles);
+  const setCurrentProfile = useAppStore((state) => state.setCurrentProfile);
+  const setProfileId = useAppPersistStore((state) => state.setProfileId);
+  const currentProfile = useAppStore((state) => state.currentProfile);
+
+  const closeDrawer = () => {
+    document.getElementById('mobile-drawer')?.classList.add('hidden');
+  };
+
+  return (
+    <div
+      id="mobile-drawer"
+      className="w-full z-[10] hidden overflow-y-auto no-scrollbar bottom-0 top-0 h-full bg-gray-100 dark:bg-black py-4 md:hidden fixed"
```

Comment:
```suggestion
      className="w-full z-10 hidden overflow-y-auto no-scrollbar bottom-0 top-0 h-full bg-gray-100 dark:bg-black py-4 md:hidden fixed"
```

z-10 is there right already?

Reply:
It is already `hidden` then why `md:hidden` again